sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -53,6 +53,7 @@ export const FastFrameTemplate = html<FastFrame>`
                         <div class="content-control-container" >
                             <label for="background-color-pickers">Background color</label>
                             <fast-radio-group
+                                tabindex="0"
```

Comment:
@marjonlynch can you advise as to why this might be necessary?

Reply:
Mark and I synced on this and the correct approach here is adding the tabIndex on the item and not the group.

Reply:
I created an issue #3300 to fix this in radio-group but maybe Kham should go ahead with what he has here until that's fixed.